### DISCUSSION TITLE
Does `fets` module export `File`?

### DISCUSSION THREAD

**Discussion by rajibkuet07** [ardatan/feTS#949] [Q&A] [answered]
[In the documentation](https://the-guild.dev/openapi/fets/client/request-params#form-data--file-uploads) `import { File, FormData } from 'fets'` is used. But in my code it shows `File` is not an exported member of `fets`.

Also, multipart form-data post request ends up with an TypeError -
`TypeError: Cannot set property name of #<File> which has only a getter`
at `getNormalizedFile` (node_modules\@whatwg-node\node-fetch\cjs\FormData.js:120:15)

Sudo Code -

```
client['/test-endpoint'].post({
  formData: {
    name: "test-name",
    avatar: new File(
      [fs.readFileSync('path/to/image-1.png')],
      `test-1.png`,
      { type: 'image/png' }
    )
  },
})
```

**Comment by ardatan** [comment]
Native `File` support added with https://github.com/ardatan/whatwg-node/commit/f844a4da5ddc8888b105b509fee7794c6217fd98
Until this gets updated in feTS, you can use `File` from `@whatwg-node/fetch`.

**Reply by rajibkuet07** [reply, marked as answer]
OK, Thanks.
